backend/development.ini.sample

```diff
@@ -861,7 +861,7 @@ caldav.radicale.headers.Access-Control-Expose-Headers = Etag
 ## Translation service target languages that will be proposed in Tracim's interface
 # It is a comma-separated list of <ISO-639-1 language code>:<Language string to display> pairs.
 # The string can use any UTF-8 character.
-; translation_service.target_languages = en:English,fr:Français,pt:Português,de:Deutsch,ar:العربية,es:Español
+; translation_service.target_languages = en:English,fr:Français,pt:Português,de:Deutsch,ar:العربية,es:Español,nb_NO:Norsk
```

Comment:
Do you sure, systran can translate this language ?

Reply:
According to https://www.systransoft.com/download/product-datasheets/systran-languages-datasheet-0120.pdf, yes.

Reply:
ok.